tpdatasrc/tpgamefiles/rules/feats/ki blast.txt

```diff
@@ -0,0 +1,10 @@
+name: Ki Blast
```

Comment:
At long last, ToEE has kamehamehas 🤣 

Only thing that's missing are EWGFs 😁 
https://m.youtube.com/watch?v=jWcAwn5LYDo

Reply:
@DudeMcDude
I'd like to have a feedback on a design choice. As the Strikes will not get fired every combat round, I decided not to use an arg as activate flag and instead worked with an effect condition. This would avoid unnecessary onBeginRound and OnDealingDamage Hooks every round or damage step. Now that I know that there is actually a limit for conditions, I am not sure what's better, Avoid unnecessarcy hooks every round and use helper conditions or avoid extra conditions. If you prefer activation flags over helper conditions, I would change the implementation.

Reply:
Yiu can assume that the condition limit will be overcome.